dask/dataframe/methods.py

```diff
@@ -351,7 +355,13 @@ def pivot_count(df, index, columns, values):
 
 
 def concat(
-    dfs, axis=0, join="outer", uniform=False, filter_warning=True, ignore_index=False
+    dfs,
+    axis=0,
+    join="outer",
+    uniform=False,
+    filter_warning=True,
+    ignore_index=False,
+    **kwargs
```

Comment:
What's the motivation for these changes?

Reply:
This was to be able to send in additional arguments like `sort=False/True` with flexibility for the caller to choose.